stix_shifter/stix_translation/src/modules/guardium/json/to_stix_map.json

```diff
@@ -0,0 +1,163 @@
+{
+  "Login Succeeded": {
+    "key": "number_observed",
+    "cybox": false,
+    "transformer": "ToInteger"
+  },
+  "Client IP": [{
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "src_ip"
+    }
+  ],
+  "Server IP": [{
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "network-traffic.dst_ref",
```

Comment:
If you are creating a network traffic object, you need to also include the protocol. Do you know if or how the protocol comes back in the Guardium results?

Reply:
Network protocol option is an optional attribute in Guardium results.

Reply:
Removed - Network protocol from json